app/src/main/java/org/greenstand/android/TreeTracker/view/SelectableItemDetail.kt

```diff
@@ -39,14 +41,25 @@ fun SelectableImageDetail(
         .padding(bottom = 12.dp) // Bottom side can be clipped by the button.
         .fillMaxWidth()
         .wrapContentHeight()
+    val userButtonModifier = Modifier
+        .padding(8.dp)
+        .width(156.dp)
+        .clip(RoundedCornerShape(10.dp))
+        .wrapContentHeight()
+
     DepthButton(
         onClick = onClick,
         isSelected = isSelected,
-        modifier = Modifier
-            .padding(8.dp)
-            .width(156.dp)
-            .clip(RoundedCornerShape(10.dp))
-            .wrapContentHeight(),
+        modifier = if (isSelected) userButtonModifier
+            .border(
+                width = 1.dp, brush = Brush.verticalGradient(
+                    colors = listOf(
+                        AppColors.Gray,
+                        AppColors.Green
```

Comment:
I think it might make more sense to use solid green, no gradient, or use a darker green instead of gray.